client/ayon_houdini/startup/otls/ayon_lop_mute_layers.hda/ayon_8_8Lop_1mute__layers_8_81.0/Sections.list

```diff
@@ -0,0 +1,10 @@
+""
+DialogScript	DialogScript
+CreateScript	CreateScript
+InternalFileOptions	InternalFileOptions
+Contents.gz	Contents.gz
+TypePropertiesOptions	TypePropertiesOptions
+Help	Help
+Tools.shelf	Tools.shelf
+IconImage	IconImage
+ExtraFileOptions	ExtraFileOptions
```

Comment:
```suggestion
ExtraFileOptions	ExtraFileOptions
AYON__icon.png	AYON_icon.png
```

Reply:
Oh nice - this makes me wonder whether it's possible to set up 1 icon that we can share across all the HDAs instead of having to have the png in every single one of them. I guess something for another PR.